armi/reactor/converters/uniformMesh.py

```diff
@@ -85,6 +86,321 @@ def converterFactory(globalFluxOptions):
         )
 
 
+class UniformMeshGenerator:
+    """
+    The :py:class:`UniformMeshGenerator <armi.reactor.converters.UniformMeshGenerator>`
```

Comment:
I don't think we need to say the name of the class in the class docstring.  Redundant.